assignment.rb

```diff
@@ -0,0 +1,97 @@
+# 1.2-Countries
+
+puts %{
+  Welcome to the file parser and formatter.
+
+  Please input the name of the countries file
+  currently stored in the same folder as this 
+  script.
+
+}
+print ">> "
+file_name = gets.chomp
+
+'''
+# For testing purposes, creating an array with special cases
+all_lines = ["MK|macedonia, the former yugoslav republic of", "FK|falkland islands (malvinas)"]
+'''
+# Create an array with the words that should not be capitalized
+lowercase_words = ['and', 'of', 'the']
+
+
+puts "Now attempting to open file: #{file_name}\n\n"
+
+# Create an empty array to hold all of the lines, then push each line in the file to it
+all_lines = []
+File.open(file_name).each do |line|
+  all_lines.push(line)
+end
+
+puts "Processing the file\n\n"
+
+all_index = 0
+
+# Perform the following logic on each line
+all_lines.each do |line|
```

Comment:
instead of requiring `all_index`, check out [this method](http://ruby-doc.org/core-2.2.2/Enumerable.html#method-i-each_with_index)